exonum/src/encoding/protobuf/proto/tests.proto

```diff
@@ -0,0 +1,55 @@
+// Copyright 2018 The Exonum Team
```

Comment:
If this file is used to test `exonum_derive`, it should be placed in `exonum_derive`, IMO. (Perhaps, as a separate crate, as it seems that cargo doesn't support build scripts for tests / examples.)